### ISSUE TITLE
BTE error on Workflow A.7

### ISSUE THREAD

**Issue by vgardner-renci** [NCATSTranslator/minihackathons#157]
https://arax.ncats.io/?source=ARS&id=16cc3949-368d-485e-a7e6-c04b8ff81050

EGFR

![image](https://user-images.githubusercontent.com/83781286/125843996-380d3607-aa4b-4479-8947-9a56f104fcb3.png)

**Comment by vgardner-renci** [comment]
```
{
  "callback": "http://0.0.0.0:8000/ars/api/messages/71e190b7-b63e-4f35-90a0-c4edbdb272d1",
  "max_results": 100,
  "message": {
    "query_graph": {
      "edges": {
        "e0": {
          "object": "n0",
          "predicates": [
            "biolink:related_to"
          ],
          "subject": "n1"
        }
      },
      "nodes": {
        "n0": {
          "categories": [
            "biolink:ChemicalSubstance"
          ],
          "name": "Chemical Substance"
        },
        "n1": {
          "ids": [
            "NCBIGene:1956"
          ],
          "name": "EGFR"
        }
      }
    }
  },
  "validation_result": {
    "message": "",
    "size": "1 kB",
    "status": "PASS",
    "version": "1.1.1"
  }
}

**Comment by newgene** [comment]
Verified it's working now, BTE returns non-empty results. It is a relatively long query.

**Comment by vgardner-renci** [comment]
@newgene Reran ARAX. Still seeing "Error / NA" for BTE.

**Comment by newgene** [comment]
@vgardner-renci similar to other issues recently closed (#155, #159, #160) after another rerun, can you give it another try?

**Comment by andrewsu** [comment]
Successfully completed with 2121 results https://arax.ncats.io/?source=ARS&id=5a5e0fef-43ba-4e48-b18d-d2d510541b64

![image](https://user-images.githubusercontent.com/2635409/128252422-3a0f0510-f997-4117-8b17-7e835fe9a1c7.png)
